main.py

```
@@ -118,18 +118,18 @@ def __init__(self):
         self.hungerbar = ""
         self.effects = {}
         self.radiation = 0
-        self.radiationbar = "\U00002622"
+        self.radiationbar = "Radiation:"
         for i in range(math.ceil(self.radiation / 100)):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)